site/write_ups/2022/02-11-22_writeup.md

```diff
@@ -0,0 +1,98 @@
+
+It took the covid-19 pandemic to push this to the point of acknowledge, but despite this being an established body of research, many (particuarly those not in minorities) were surprised to learn of this inequality. We discussed this surprise, what responsibility the public, the media and those working in the field have to report and acknowledge these inequalities and potential consequences of the way we record race.
+
+## Did you know the maternal mortality rates discussed in the article? Were you surprised?
+
+Some of us were unaware of this, especially pre-covid but came to be aware of it during covid. Many of us were vaguely aware before but seeing the scale of the difference was shocking. Some knew and weren't surprised due to personal experiences or were aware through friends who have personal experience/insight such as midwives, doctors or black mothers. Some famous people such as Serena Williams have already [spoken publicly about her experiences of racism in healthcare](https://www.bbc.co.uk/sport/tennis/43299147). This is depressing but perhaps not surprising, with medicine, examples, and diagrams/explanations geared/aimed at towards white men. 
+
+Research from 1970s has shown black males having disproportionate restraints/drug doses in mental health care. Why has nothing been done about this? Stories stick around longer even when they are disproved with research - the example of vitamin D was mentioned.
+
+Many of us were aware of the research bias. For example; how psychological research studies are done on psychology students or how medical text books show e.g. skin conditions on white people or autism diagnoses. Even within biology (for example reproductive biology), people thought of how important sperm was until female scientists came along, then they thought about the eggs
+ https://web.stanford.edu/~eckert/PDF/Martin1991.pdf  - The Egg and the Sperm: How Science Has Constructed a Romance Based on Stereotypical Male-Female Roles. Article - https://bigthink.com/surprising-science/there-is-inherent-sexism-in-how-we-talk-about-biology/
+
+60% of people who died from covid were disabled - this was disproportionate even when those disabilities were nothing to do with respitory diseases e.g. lung function/immune system. This is partially because anyone with a learning disabilities had a do not rescucitate order as a default. Some of these people were on the one hand "essential workers" and on the other hand disposable. And there was a similar talking point to the one in the article, "genetics must be causing this issue", in the media with the "underlying health conditions" framing.
```

Comment:
```suggestion
60% of people who died from covid were disabled - this was disproportionate even when those disabilities were nothing to do with respiratory diseases e.g. lung function/immune system. This is partially because anyone with a learning disability had a do not resuscitate order as a default. Some of these people were on the one hand "essential workers" and on the other hand disposable. And there was a similar talking point to the one in the article, "genetics must be causing this issue", in the media with the "underlying health conditions" framing.
```